scripts/actions/mobskills/pirate_pummel.lua

```diff
@@ -0,0 +1,27 @@
+-----------------------------------
+-- Pirate Pummel (Lion)
+-- Damage and burn effect
+-- Type: Physical
+-- Skillchain Properties: Fusion/Impaction
+-----------------------------------
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    local numhits = 1
```

Comment:
Looking at pirate pummel on retail, I'd be very surprised if it was single-hit given the animation, but I don't know how to confirm that - action packets?

Reply:
added a TODO for this in the script as I do not have access to this trust in retail.

Reply:
@zach2good it its not visible in packet someone with retail access might be able to verify by getting beaten up with one of the BCNM featuring her, assuming its the same as the trusts version (I would think at least hit count would be)